packages/utility/agent.ts

```diff
@@ -138,6 +138,7 @@ const User = {
 
 const Comments = {
   put: (id: string, body: {}) => requests.put<Comment>(id, body),
+  post: (body: {}) => requests.post<Comment>("comments", body),
```

Comment:
why the body is typed as `{}`? I think it should be `(body: Comment) => requests.post<Comment>("comments", body)`